src/stactools/nisar_sim/commands.py

```diff
@@ -37,20 +38,26 @@ def create_collection_command(destination: str) -> None:
 
         return None
 
-    @ephemeralcmd.command("create-item", short_help="Create a STAC item")
+    @nisarsim.command("create-item", short_help="Create a STAC item")
     @click.argument("source")
     @click.argument("destination")
-    def create_item_command(source: str, destination: str) -> None:
+    @click.argument("dither")
```

Comment:
Should this be an option? so we can list the choices, and there is a sensible default set? Also needs to be in the Doc String for user feedback on the CLI.

Reply:
"X" is the most reasonable dither option meaning no dither. Changing this to a @click.option with default X